### PR TITLE
Added postgres docker scripts for local dev & instructions for use.

### PR DESCRIPTION
**Pull request by stephenhand**
* Added postgres docker scripts to allow local dev without needing to install postgres locally
* Allowed Basic auth to be enabled via an environment variable
* Added script for more modern sample data.

Basically a few updates & enhancements that will go alongside updates to the local onboarding docs that should make life easier for new starters (and existing devs).

Primary reviewer: @murilovmachado

### CODE REVIEW

**Review by GPaoloni — APPROVED** [review]
This is awesome! Thanks!
Small question: can we use this to run the test suits?

File: app.js [hunk 1]
```diff
@@ -115,10 +115,13 @@ async function authorizationMiddleware(req, res, next) {
  * NOTE: If we ever move this project to Typescript: https://dev.to/kwabenberko/extend-express-s-request-object-with-typescript-declaration-merging-1nn5
  */
 const addAccountSid = (req, res, next) => {
+  console.log('accounts SID middleware');
```
[the remark this answers is not in the document]
Comment (stephenhand) [reply app.js]: You are correct, the extra logs were just for local debugging and shouldn't be in the PR, removed them now.

**Comment by stephenhand** [comment]
> 

> This is awesome! Thanks! Small question: can we use this to run the test suits?

The docker database? Yes, as long as docker is available in the test environment you can use it - testing is a good use case for it

File: app.js [hunk 2]
```diff
@@ -73,7 +73,7 @@ async function authorizationMiddleware(req, res, next) {
   }
 
   if (authorization.startsWith('Basic')) {
-    if (process.env.NODE_ENV === 'test') {
+    if (process.env.NODE_ENV === 'test' || process.env.HRM_PERMIT_BASIC_AUTHENTICATIO) {
```
Comment (nickhurlburt) [inline app.js:76]: Is there a reason (like a 30-character limit) to leave off the `N` in `AUTHENTICATION`?

**Review by nickhurlburt — APPROVED** [review]
This is great. We've been behind on using docker for local environments and this will be a good start.

File: app.js (hunk 2 above)
Comment (stephenhand) [reply app.js]: No, I renamed the variable at the last minute to be more inline with existing stuff, must've introduced a typo

**Pull request merged**